Makefile

```diff
@@ -23,3 +24,14 @@ test:
 .PHONY: lint
 lint:
 	docker run --rm -v $$(pwd):/app -w /app golangci/golangci-lint:v1.46.2 golangci-lint run -v
+
+# Builds multiple architectures at once.
+# Requires docker buildx and QEMU to be configured.
+# Because of how docker buildx works, in that it _must_ push when it builds, push is the default of this task.
```

Comment:
I'm not sure I understand why a push is required. Why do we want our build command to automatically push the image?

Reply:
This is a nuance of how `buildx` works. It doesn't allow for building first, then pushing later, so, if you want to push the image that is produced by the multi-arch build, it has to be done at build time with this flag.

Reply:
Ultimately, this `buildx` tasks should be what is used for CI/CD to build and push multi-arch images, and the old `build` and `push` tasks can be used for testing / development things.